rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -534,6 +534,7 @@ impl Controller {
         )?;
         let connlib = connlib_client_shared::Session::connect(
             login,
+            Sockets::new()?,
```

Comment:
That #2986 would be really nice here for a default.

Reply:
Great minds think alike, that is why I reopened it 😁